README.md

```diff
@@ -1,6 +1,15 @@
 # Enterprise Course Catalog: OPENLXP-XDS
 
 
+## Authentication
+
+The enivronment variables `SP_PUBLIC_CERT`, `SP_PRIVATE_KEY` , and `SP_ENTITY_ID` must be defined (if using docker-compose the variables can be passed through).
```

Comment:
Is it possible (and is it a good idea) to update the authentication package to be able to manage these attributes via the Django admin?

Reply:
We definitely do not want to store SP_PRIVATE_KEY in the database.  That is very bad practice.  

Ideally we would use one of the methods listed [here](https://security.stackexchange.com/questions/12332/where-to-store-a-server-side-encryption-key), but can't really guarantee being able to perform any of them.

Docker supports storing sensitive info in [secrets](https://docs.docker.com/engine/swarm/secrets/), but that is only available with Docker Swarm.

We could switch from environment variables to mounting the cert and key as files to the docker container and then reading the files in to set the setting.

Reply:
Makes sense -- I don't think we need to change the implementation.  Instead we could create a ticket for the backlog which would be to support this option at some point in the future.